### PR TITLE
[EN/RU] Update BanchoBot, Beatmap Editor, and How You Can Help

### PR DESCRIPTION
**Pull request by TicClick**

### CODE REVIEW

**Comment by TicClick** [comment]
> How you can help is outdated, use the English version for reference.
I know, that's why I've updated the translation accordingly and written some parts of the Russian article myself.

**Comment by sr229** [comment]
Checking the contents, it still uses the peppy post version which is outdated (correct me if I'm wrong), the more up to date and up to standards version is the one you should be basing off IMO. 


Let me know if that's not the case.

**Comment by TicClick** [comment]
I admit it's misleading, yet the very first sentence reads: "This article is based off one of peppy's posts: ...". I've found it useful to leave some links for backtracking or attributing, like "this stems from that", or "As seen on ...".

While this very foreword belongs to the initial version of the article, the contents differ drastically, so no, I went ahead and put some time and effort into skimming through pull requests/sprints to ensure no one plans to deprecate the article, and then making a fresh off-the-hook translation mixed with some good write-up manner thoughts.

**Comment by cl8n** [comment]
if your changes differ from the current English article , could you also update English accordingly? that way we can review the new content and have it match between languages

**Comment by TicClick** [comment]
Although that is a good point, I have a few objections:
1) It surely belongs to a different changeset -- all I'm currently up for is translations.
2) Articles such as community-, not game mechanics-oriented, don't necessarily have to match between different languages, as different sub-communities have different, well, mindsets, and different features or details to outline. The overall tone of the article also matters.
3) I may or may not have skills deemed necessary for article building on a foreign language. While I hope I do, a certain community-driven knowledge base doesn't have to suffer from my experiments on English.

File: wiki/How_You_Can_Help!/ru.md [hunk 1]
```diff
@@ -1,106 +1,97 @@
...
-outdated: true
----
+*Прим. перев.: эта статья основана на одном из постов peppy под названием [How you can help!](/community/forums/topics/83155).*
 
-# Как можно помочь
+# Чем помочь osu!
 
-_Это — перевод поста, который как-то раз взял и написал, а потом попросил превратить в статью сам [peppy](https://osu.ppy.sh/users/2 "peppy"). Оригинал можно найти на форуме: [How you can help!](https://osu.ppy.sh/community/forums/topics/83155)._
+Как и любая онлайн-игра, osu! объединяет самых разных людей со всего света, и в глазах многих из них горит огонь.
+Но как же его применить?
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: так, про огонь в глазах я уже говорила 
замени на "среди них много энтузиастов" или что-то в этом роде!

**Comment by cl8n** [comment]
you're doing more than translating, you're also changing the meaning of the article. that's okay, as long as English doesn't become out-of-dateand it's fine to have the articles differ if there are differences between the languages' communities. the problem is that I've got no way to check besides using machine translations (which don't usually preserve tone/style). this is one of the reasons we generally avoid adding new content in translations without first updating English On Oct 2, 2019 13:14, TicClick <notifications@github.com> wrote:Although that is a good point, I have a few objections:
It surely belongs to a different changeset -- all I'm currently up for is translations.Articles such as community-, not game mechanics-oriented, don't necessarily have to match between different languages, as different sub-communities have different, well, mindsets, and different features or details to outline. The overall tone of the article also matters.I may or may not have skills deemed necessary for article building on a foreign language. While I hope I do, a certain community-driven knowledge base doesn't have to suffer from my experiments on English.

File: wiki/How_You_Can_Help!/ru.md [hunk 2]
```diff
@@ -1,106 +1,97 @@
...
 
-Как мне кажется, я целиком занят.
-Всё свободное время я трачу на osu! или osu!stream и больше никак не могу в них вложиться.
-Это значит, что почти всё, что запланировано, делается в соответствии с моими личными приоритетами, — но не значит, что для принесения пользы нужно быть разработчиком; возможностей очень много!
+## Правка и перевод вики
 
-Итак, перед вами список, начинающийся с самых важных дел, где, как я думаю, может помочь кто угодно.
-Я постараюсь его обновлять по мере появления новых дел и обязанностей, но вы тоже можете предлагать свои правки.
+[Как говорит нам Wikipedia](https://ru.wikipedia.org/wiki/%D0%92%D0%B8%D0%BA%D0%B8), вики — это база знаний, которую может редактировать кто угодно.
+У osu! [тоже есть своя вики](https://github.com/ppy/osu-wiki) — например, именно на ней расположена статья, которую вы читаете прямо сейчас.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: я думаю, что я и так поняла бы на какой странице я читаю статью лол

File: wiki/How_You_Can_Help!/ru.md [hunk 3]
```diff
@@ -1,106 +1,97 @@
...
-  Нам нужна помощь всего сообщества, не только его англоговорящей части (занимательный факт: более 75% пользователей плохо понимают английский или не знают его совсем).
+## Помощь модераторам
 
-## Поддерживайте osu! в чистоте!
+У osu! есть своя команда модераторов, следящих за сообществом, будь то форум, чат или комментарии к картам.
+Однако, как и в любом культурном обществе, его участники проактивны: они не только полагаются на чужую помощь, но и сами стараются поддерживать порядок.
 
-В osu! есть модераторы, но за потоком активности в 25+ каналах — и тем более на форумах — трудно пристально следить.
-К счастью, это легко исправить.
+Сделать это достаточно легко: если вы где-то заметили ошибку или нарушение [правил osu!](/wiki/Rules), сообщите о нём модераторам с помощью [чат-команды `!report`](/wiki/Reporting_Bad_Behaviour), и справедливость будет восстановлена, а ошибка — исправлена.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: ошибку и нарушение -> сообщите о нем
я понимаю, конечно, что речь про нарушение
но, может, лучше заменить на "об этом"?

File: wiki/How_You_Can_Help!/ru.md [hunk 4]
```diff
@@ -1,106 +1,97 @@
...
+Перемотайте её в конец, и...
 
-- Выберите один из этих форумов: [Техподдержка](https://osu.ppy.sh/community/forums/5), [Помощь с маппингом](https://osu.ppy.sh/community/forums/56), [Сторибординг](https://osu.ppy.sh/community/forums/20), [Геймплей и ранкинги](https://osu.ppy.sh/community/forums/13).
-- Ткните в ссылку `Mark as read` в правом верхнем углу.
-- Следите за новыми темами (они помечены яркими иконками) и по возможности туда отвечайте.
-- Читайте [руководства на вики](/wiki/List_of_Guides "Список руководств") и просвещайтесь.
-  Допишите их, если чувствуете в себе силы.
+Процесс помощи сводится к трём простым шагам:
+1. Нажмите на кнопку `Mark as read`.
+2. Следите за новыми темами (теперь они будут помечены яркими иконками) и по возможности туда отвечайте.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: про иконки не совсем понятно, если честно

File: wiki/How_You_Can_Help!/ru.md [hunk 5]
```diff
@@ -1,106 +1,97 @@
...
 
-## Проверяйте и модьте карты!
 
-Несмотря на то, что для ранкинга существуют отдельные группы людей, помогать может любой желающий.
-Пишите отзывы к опубликованным картам, оставляйте конструктивную критику, и, возможно, когда-нибудь вы попадёте в одну из тех самых групп!
+## Моддинг карт
 
-### Откуда начать
+osu! так устроено, что карты, которые вы играете, проверяются и тестируются живыми людьми, и написать полезный комментарий, а то и несколько, может любой игрок.
+Самые внимательные и опытные комментаторы получают право номинировать карты на ранк и тем самым влиять на многообразие музыки в osu!.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: мне не нравится последнее предложение
оно звучит так, как будто активные люди стопроцентно получают право номинировать мапки
и влияют все же не комментирующие игроки на многообразие музыки, а мапперы 
даже если карта не будет ранкнута например, она же все равно на сайте будет

File: wiki/BanchoBot/ru.md [hunk 6]
```diff
@@ -1,35 +1,153 @@
...
+
+*Примечание: параметр `<пользователь>` нечувствителен к регистру.*
+
+### HELP
+
+```
+!HELP
+```
+
+Показать списк доступных команд для BanchoBot.
```
Comment (ryuu) [inline wiki/BanchoBot/ru.md]: спис**о**к

File: wiki/How_You_Can_Help!/ru.md [hunk 7]
```diff
@@ -1,106 +1,97 @@
...
-- Исследуйте [список опубликованных карт](https://osu.ppy.sh/beatmapsets?s=pending) по любимым артистам или песням (или выберите карту наугад).
-- Сыграйте карту и напишите в тред на форуме свои впечатления.
-- Ознакомьтесь с [руководством по моддингу](/wiki/Modding "Моддинг") и выясните, на что нужно делать упор и обращать внимание.
-- Прочтите [Критерии ранкинга](/wiki/Ranking_Criteria "Критерии ранкинга") и запомните основные моменты.
-- Пользуйтесь каналами `#modhelp` и `#modreqs`; писать необязательно, можно и просто читать.
-- Если у вас много свободного времени, создайте очередь моддинга в [соответствующем форуме](https://osu.ppy.sh/community/forums/60) и пишите моды.
+- Исследуйте [список опубликованных карт](/beatmapsets?s=pending) по любимым артистам или песням (или выберите карту наугад);
+- Сыграйте понравившуюся карту и оставьте свои впечатления на странице для обсуждений;
+- Хотите пойти дальше? Ознакомьтесь с [руководством по моддингу](/wiki/Modding) и выясните, на что нужно делать упор и обращать внимание;
+- Прочтите [критерии ранкинга](/wiki/Ranking_Criteria) и запомните основные моменты и критические ошибки;
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: блин, опять твое "прочтите" ;__;
про-чи-тай-те

File: wiki/How_You_Can_Help!/ru.md [hunk 8]
```diff
@@ -1,106 +1,97 @@
...
+- Пользуйтесь каналами `#modhelp` и `#modreqs` (писать необязательно, можно и просто читать);
+- Если у вас много свободного времени, создайте очередь моддинга в [соответствующем форуме](/community/forums/60) и пишите моды на заказ.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: очередь моддинга, пишите моды
замени

File: wiki/How_You_Can_Help!/ru.md [hunk 9]
```diff
@@ -1,106 +1,97 @@
...
 
-## Работайте над всякими прикольными проектами вместе с другими!
 
-Вокруг osu! существует много различных проектов и идей: турниры, активности, связанные с маппингом, различные неофициальные приложения, ... . Вы можете помочь их авторам при наличии требуемых навыков.
+## Всякие прикольные идеи
 
-### Откуда начать
+Вокруг osu!, как вокруг огромной звезды, вращается много проектов, начиная с турниров и заканчивая неофициальными, но полезными приложениями.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: мне кажется, что изначальный вариант с перечислением был лучше 
без того, что после троеточий
не идеальный, но все же

File: wiki/How_You_Can_Help!/ru.md [hunk 10]
```diff
@@ -1,106 +1,97 @@
...
-- Загляните в [турнирный форум](https://osu.ppy.sh/community/forums/55).
-  Возможно, вы придётесь кстати, например, выступите в качестве зрителя или судьи.
-- Исследуйте [форум по разработке](https://osu.ppy.sh/community/forums/2).
-  Активность там низкая, но [проекты](https://osu.ppy.sh/community/forums/topics/58855) [более](https://osu.ppy.sh/community/forums/topics/80027) [серьёзны](https://osu.ppy.sh/community/forums/topics/79354) и масштабны.
-- Влейтесь в какой-нибудь [проект по маппингу](https://osu.ppy.sh/community/forums/53) и замутите с другими мапперами коллаб.
-- Если у вас есть классная идея — начните свой проект!
-  Если требуется помощь или совет, можете попробовать написать мне (тут есть шанс, что я не отвечу, если у меня много дел).
-- Оживите [osu!monthly](https://osu.ppy.sh/community/forums/topics/75017), я без него скучаю :<
+- Загляните в [турнирный форум](/community/forums/55).
+  Возможно, вы придётесь кстати, например, выступите в качестве зрителя или судьи;
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: возможно, кстати, например

File: wiki/How_You_Can_Help!/ru.md [hunk 11]
```diff
@@ -1,106 +1,97 @@
...
+- Исследуйте [форум разработчиков](/community/forums/2).
+  Активность там низкая, но [проекты](/community/forums/topics/58855) [более](/community/forums/topics/80027) [серьёзны](/community/forums/topics/79354) и масштабны;
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: чьи проекты-то? **их**, **их** проекты!

File: wiki/How_You_Can_Help!/ru.md [hunk 12]
```diff
@@ -1,106 +1,97 @@
...
+- Влейтесь в какой-нибудь [проект по маппингу](/community/forums/53) и замутите с другими мапперами совместную карту — [коллаб](wiki/Guides/Collab_Information);
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: убери "замутите"

**Comment by TicClick** [comment]
It's more about rewording (a large portion of, oftentimes) rather than appending something new: there's always a more elegant way to say something while preserving the original thought or idea, which is probably the best description of what I've done here.

Take "How you can help!", for example. It's got a section for every direction one can help on, with all the details sparkling and gleaming about which subforums you may visit to provide your help, or the general steps you need to take to help the chat moderators look after the conversations.

During translation, I've kept the general flow with all the links and steps intact, yet I've used different sentence schemas, different, I don't know, preamble and epilogue, perhaps, and maybe reordered so,mething here and there, while adding a link to Wikipedia for better explanation. I see all these as changes acceptable to be applied to a translation only -- there's no need to butcher the original as long as it all has the same meaning and article structure, which the translations try to adhere to.

On the contrary, had I found a new way to contribute to osu!, I woulda changed both articles, be it a single PR or not.

File: wiki/How_You_Can_Help!/ru.md [hunk 13]
```diff
@@ -1,106 +1,97 @@
...
 
-Я каждый день вижу в osu! какую-то грязь.
-Люди жалуются друг на друга, на читеров, на фармеров, на изменения и новизну, на весь белый свет, на что угодно.
-Разумеется, нам всем ещё очень далеко до уровня комментариев на ютубе, но это не значит, что ничего нельзя улучшить.
+В 2017 peppy и его команда начали работать над [osu!lazer](https://github.com/ppy/osu) — новым клиентом игры — и [переписывать сайт](https://github.com/ppy/osu-web) на более современные технологии, выложив и то, и другое в [открытый доступ](https://ru.wikipedia.org/wiki/%D0%9E%D1%82%D0%BA%D1%80%D1%8B%D1%82%D0%BE%D0%B5_%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%BD%D0%BE%D0%B5_%D0%BE%D0%B1%D0%B5%D1%81%D0%BF%D0%B5%D1%87%D0%B5%D0%BD%D0%B8%D0%B5).
 
-### Откуда начать
+Чтобы будущее наступило быстрее, peppy попросил о помощи других программистов, дизайнеров, игроков — и они откликнулись!
+И продолжают вносить свой вклад до сих пор.
+Возможно, и вы окажетесь в их числе?
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: мне не нравится, как это выглядит
второе предложение про вклад выглядит отрезанным
вот такие у меня придирки странные

File: wiki/How_You_Can_Help!/ru.md [hunk 14]
```diff
@@ -1,106 +1,97 @@
...
 
-- Будьте дружелюбны к остальным: жизнь слишком коротка, чтобы забивать её негативом.
+
+## Счастье всем и каждому
+
+Разумеется, нельзя помочь *всем и везде,* но зато можно — *хоть кому-то и хоть с чем-то.*
+Жизнь слишком коротка, чтобы ссориться и терпеть негатив, и вряд ли кто-то по-настоящему хочет оказаться среди плохих или бестолковых людей.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: бестолковые -- слишком грубо, скорее ... бестактные? не знаю
замени, пожалуйста

File: wiki/How_You_Can_Help!/ru.md [hunk 15]
```diff
@@ -1,106 +1,97 @@
...
+Любой, даже самый крохотный вклад, несёт пользу.
+
+- Будьте если не дружелюбны, то нейтральны к остальным;
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: будьте дружелюбны, или хотя бы нейтральны к остальным

File: wiki/How_You_Can_Help!/ru.md [hunk 16]
```diff
@@ -1,106 +1,97 @@
...
 - Не жалуйтесь на читеров на публике.
-  [Напишите на них репорт](/wiki/Reporting_Bad_Behaviour/Handling_Cheating "Реагирование на читеров") и занимайтесь дальше своими делами: рано или поздно все нечестно сыгранные результаты будут стёрты.
+  [Напишите на них жалобу](/wiki/Reporting_Bad_Behaviour/Handling_Cheating) и занимайтесь дальше своими делами: рано или поздно все нечестно сыгранные результаты будут стёрты;
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: "на них" тут не нужно

File: wiki/How_You_Can_Help!/ru.md [hunk 17]
```diff
@@ -1,106 +1,97 @@
...
 - Поддерживайте других людей и помогайте им в разрешении негативных и конфликтных ситуаций.
+  Ладно, можете не помогать, но хотя бы не разжигайте!
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: ну и зачем тогда статья? ну типа
ну вот вы прочитали
забейте кароч
можете не помогать
перепиши

**Comment by TicClick** [comment]
What I want to express is: a translation that performs a line-to-line mapping works nice when applied in situations where you need to keep not only the meaning, but the wording itself as well (re: dialogues/scripts, some of interfaces, step-to-step guides, novels/books in general).

If you're only here to pass knowledge, well, you don\'t need to worry about the medium too much and are free to use whatever words and/or phrases fit the best.

File: wiki/How_You_Can_Help!/ru.md (hunk 4 above)
Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: переписал без иконок (amen)

File: wiki/How_You_Can_Help!/ru.md (hunk 5 above)
Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: убрал, переписал на абстрактное «карты будут лучше»

File: wiki/How_You_Can_Help!/ru.md (hunk 11 above)
Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: необязательно, но пускай будет так

File: wiki/How_You_Can_Help!/ru.md (hunk 13 above)
Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: нет, всё верно

File: wiki/How_You_Can_Help!/ru.md (hunk 17 above)
Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: сделал это предложение менее подстрекающим

File: wiki/How_You_Can_Help!/ru.md [hunk 18]
```diff
@@ -78,20 +78,19 @@ osu! так устроено, что карты, которые вы играе
...
-И продолжают вносить свой вклад до сих пор.
-Возможно, и вы окажетесь в их числе?
+Возможно, вы тоже окажетесь в их числе и внесёте свой вклад?
 
 
 ## Счастье всем и каждому
 
 Разумеется, нельзя помочь *всем и везде,* но зато можно — *хоть кому-то и хоть с чем-то.*
-Жизнь слишком коротка, чтобы ссориться и терпеть негатив, и вряд ли кто-то по-настоящему хочет оказаться среди плохих или бестолковых людей.
+Жизнь слишком коротка, чтобы ссориться и терпеть негатив, и вряд ли кто-то по-настоящему хочет оказаться среди плохих или глупых людей.
```
Comment (ryuu) [inline wiki/How_You_Can_Help!/ru.md]: **глупых**
ну я же просила

Comment (TicClick) [reply wiki/How_You_Can_Help!/ru.md]: нæт

File: wiki/How_You_Can_Help!/ru.md (hunk 13 above)
Comment (ryuu) [reply wiki/How_You_Can_Help!/ru.md]: И ВСЕ РАВНО ЗАМЕНИЛ

**Comment by cl8n** [comment]
I feel like you're missing my point; your method of translation is good
and you shouldn't be word-for-word translating English or else it'd
probably sound awkward in Russian. I agree with you there.

The problem is that you've added a bunch of content that isn't in the
English article and has nothing to do with the differences between
English and Russian. For example, this link to peppy's old forum post
before the title (which we specifically *removed* from English a while
ago...), or this definition of the word "wiki" in the first paragraph:
that's the kind of stuff you should be porting back to English. In this
case, my review would tell you to remove both of those. I can't tell if
you added anything else like this because I'd have to rely on machine
translations for the whole article and then do my best to compare it
back to English.

In general, translations should keep the same ideas as English (only
exception being where the languages' communities view something
differently or use different common references), but the actual words &
writing can be whatever works best for the language. If this isn't
written in the wiki contribution guide or ASC I'll make a note to add
it there soon

**Comment by TicClick** [comment]
Can't completely agree with you on that; although I see where you come from, the details you mentioned don't really change anything, that is, they're small enough to pass as an additional level of explanation or a detail for curious readers. Moreover, if I consider them as insignificant (and if they really are), they don't need to be propagated to the original article.

For example, does one really need to explain what a wiki is, given that the first sentence of the corresponding paragraph in English article is "The wiki is our official knowledge base about <...>"? Or is there any benefit from having one more definition of "being morally/lawfully good" in the epilogue? One can't argue that these aren't welcome in the translation as well, but again, they're the details that may ease the reading process, or make it more entertaining, or provide additional examples, or... the list goes on.

As for the post link, well... Gotta remove it, no one probably needs it anyway, and revise the whole changeset.

**Comment by cl8n** [comment]
> they're small enough to pass as an additional level of explanation or a detail for curious readers

how is this specific to Russian? if you think something adds explanation or detail, add it to English too. that way one of the wiki maintainers can review your addition and see if it fits the article. none of us can read Russian at the moment

> For example, does one really need to explain what a wiki is, given that the first sentence of the corresponding paragraph in English article is "The wiki is our official knowledge base about <...>"?

I'd say "no", which is why I don't understand why you included this in Russian when it wasn't in English in the first place. Just translate the English version...? Is there something wrong with it? (if there is something wrong with English, change it!)

**Comment by TicClick** [comment]
I don't require you to read Russian, as I'm able to scavenge Russian community for reviews and/or reviewers (gonna be a surprise if it really works out), so you shouldn't worry about this too much; besides, I suspect there are already translations not revised by natives, which sometimes results in low-quality outcome.

Of course things I've mentioned are not Russian-specific, but neither they're of any importance such that it's necessary to add them to the original article.

Again, an extra example I've put for better understanding, or a slightly more lengthly or detailed explanation of a concept belong more to the derivative work itself -- the original is good to go without them, as long as I don't introduce anything new during (re-)translation or refresh and the English article is already well-written. Some are, some are not; it's a completely different topic, considering that less than one fifth of the whole wiki is in Russian and it needs more attention that it receives as of now. If I am to enrich English versions first, this may very well double the amount of work (alright, not double, but you got it).

File: wiki/BanchoBot/ru.md [hunk 19]
```diff
@@ -1,35 +1,153 @@
...
+
+*Примечание: параметр `<пользователь>` нечувствителен к регистру.*
+
+### HELP
+
+```
+!HELP
+```
+
+Показать список доступных команд для BanchoBot.
```
Comment (AnnSur) [inline wiki/BanchoBot/ru.md]: "Показать" (как и начало описания других команд ниже) точно должно быть в инфинитиве? Ничего против не имею, можно и инфинитив оставить, но почему не "Показывает", "Выбирает" и т д?

File: wiki/BanchoBot/ru.md [hunk 20]
```diff
@@ -1,35 +1,153 @@
...
+
+### WHERE
+
+```
+!WHERE <пользователь>
+```
+
+Показать, в какой стране сейчас находится пользователь.
+
+*Примечание: будет показана только страна, без города.*
```
Comment (AnnSur) [inline wiki/BanchoBot/ru.md]: Кстати говоря, если человек указал в настройках "Показывать другим свое местоположение", то город будет тоже указывается. Не понимаю, почему в английской версии написано только про то, что будет показан только страна (но при этом город также (с учетом установки той самой настройки) может быть показан). А что если написать не "будет показана только страна", а что-то в духе "в основном показывается страна" или по-другому... ?

File: wiki/BanchoBot/ru.md (hunk 19 above)
Comment (TicClick) [reply wiki/BanchoBot/ru.md]: я исходил из того, что это команды, и формулировал мысль как «`команда` означает `сделать что-нибудь`», ты не первая, кто на это обращает внимание, поэтому сделаю получше (без инфинитива)

File: wiki/BanchoBot/ru.md (hunk 20 above)
Comment (TicClick) [reply wiki/BanchoBot/ru.md]: добавил, потом в оригинальную статью принесу

File: wiki/BanchoBot/ru.md [hunk 21]
```diff
@@ -1,35 +1,153 @@
...
+**БанчоБот** — чат-бот osu!, запрограммированный [Echo49](https://osu.ppy.sh/users/431).
+Он помогает администраторам управлять [IRC](/wiki/Internet_Relay_Chat), а пользователям показывает справку и информационные сообщения (например, о перезапуске сервера).
+Ещё у бота есть собственные профили в [osu!](https://osu.ppy.sh/users/3) и [Твиттере](https://twitter.com/banchoboat).
+
+## Команды
+
+*Вы также можете ознакомиться с [полным списком команд чата](/wiki/Chat_Console#commands-list).*
+
+Все команды, которые знает BanchoBot, начинаются с `!` и работают как внутри osu!, так и во внешних IRC-клиентах.
+Команды можно отправлять в личку BanchoBot или прямо в общий чат (в этом случае их никто, кроме отправителя, не увидит).
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/BanchoBot/ru.md]: поменял

**Comment by TicClick** [comment]
Now the articles should be more or less aligned with their translations where it is deemed necessary

**Comment by sr229** [comment]
What @cl8n said is inasmuch we want you to have it spiritually the same as the English one, we're looking for **parity** - aka, despite sentence structure changes, it should be fundamentally the same. So as a translation article you're not allowed to go beyond what the original base article has (which I would RFC as a part of criteria later since we're gonna have a bit more trouble about this).

File: wiki/Beatmap_Editor/ru.md [hunk 22]
```diff
@@ -1,22 +1,21 @@
...
--   [Спейсинг](/wiki/Beatmap_Editor/Distance_Snap)
--   [Нестандартные хитсаунды](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides)
--   [Киаи](/wiki/Beatmap_Editor/Kiai_Time)
+- [Меню](Menu) — содержащее в себе множество инструментов и настроек;
+- [AiMod](AiMod) — проверка карты на некоторые распространённые ошибки;
+- [Временная шкала](Timelines), где расположены игровые объекты и текущее положение в песне;
+- [Деления временной шкалы](Beat_Snap_Divisor), влияющие на частоту расстановки объектов;
+- [Спейсинг](Distance_Snap), отвечающий за расстояние между нотами;
+- [Киаи](Kiai_Time) — вспышки и фонтаны из звёзд для акцентирования участков песни;
+- [Нагрузка от сториборда](SB_Load), показывающая, как сильно фоновая анимация влияет на производительность игры.
```
Comment (AnnSur) [inline wiki/Beatmap_Editor/ru.md]: Я думаю, если писать через тире, то надо добавлять "это *какое-то существительное* *описание*", например в 16 строчке такая структура соблюдается, а в 15 я не вижу. 
И я думаю, было бы красивее выбрать что-то одно: 
- либо описание через тире ("[Временная шкала](Timelines) — элемент редактора (раздела "Compose") " ), 
- либо без него при помощи причастия (как в 19 строчке здесь), глагола и пр. Например "[Меню](Menu) содержит...", "[AiMod](AiMod) осуществляет проверку карт...". 
Ну или смешать и предложения с тире, и без него. 

! на счет шага временной шкалы я бы сказала то, что он влияет по идее не только на частоту расстановки объектов, но и впринципе за такт(ритмику?). Этот параметр скорее отвечает за сам такт. По идее, шаг временной шкалы - это бегунок/параметр, отвечающий за выставление такта/доли (или как-то по-другому это надо назвать все вот эти 1/3, 1/4,..6..8..12..16). Чем выше значение, тем больше тиков.

>[Киаи](Kiai_Time) — вспышки и фонтаны из звёзд для акцентирования участков песни;
- "для акцентирования" можно ли написать ",акцентирующие.."?
- "участков песни", но не прям участки песни. все же киай выделяет конкретный промежуток (припев) в песне, этот промежуток является каким-то ярким, крутым, захватывающим.   Можно ли написать "акцентирующий яркие участки песни"? или что-то в таком духе. просто по идее само слово "акцентирующий" уже говорит о том, что там какие-то особенные моменты в песне, по идее можно и не зацикливать прям на этом еще одним каким-либо словом

Comment (ryuu) [reply wiki/Beatmap_Editor/ru.md]: не соглашусь с последним, потому что можно поставить совсем маленький киаи, длящийся не больше секунды
это делает акцент на какой-то определенный громкий звук 
например финиш
или начало куплета

File: wiki/Beatmap_Editor/ru.md [hunk 23]
```diff
@@ -1,22 +1,21 @@
----
-outdated: true
----
+# Редактор карт
 
-# Beatmap Editor
+У osu! есть встроенный **редактор** для создания собственных карт. Он состоит из четырёх основных разделов:
 
-![Главный раздел редактора в режиме окна](Compose/Compose.png "Главный раздел редактора в режиме окна")
+1. [Compose](Compose): здесь создаются **паттерны** из нот и слайдеров.
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Beatmap_Editor/ru.md]: возможно, это зависит от языка

File: wiki/Beatmap_Editor/ru.md [hunk 24]
```diff
@@ -1,22 +1,21 @@
...
+2. [Design](Design): раздел для редактирования **[сториборда](/wiki/Storyboards)** — разнообразной анимации, которая проигрывается на фоне во время игры.
+  Здесь также можно добавить фоновую картинку или видео.
+3. [Timing](Timing): выставление оффсета и BPM песни, чтобы ноты попадали в ритм.
+  Кроме того, тут можно настроить **[хитсаунды](/wiki/Guides/Custom_Sample_Overrides)** — звуки ударов по игровым объектам.
+4. [Song Setup](Song_Setup): настройка игровых параметров карты (например, размера нот), а также названия песни и прочих **метаданных.**
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Beatmap_Editor/ru.md]: там же можно настроить не только нестандартные, но и стандартные хс. а кроме того, «тут/здесь» я считаю неплохим решением вполне в духе публицистики (хоть это и не она)

File: wiki/Beatmap_Editor/ru.md [hunk 25]
```diff
@@ -1,22 +1,21 @@
...
-
--   [Маппинг](/wiki/Beatmapping) (Основы маппинга)
--   [Шаг временной шкалы](/wiki/Beatmap_Editor/Beat_Snap_Divisor)
--   [Спейсинг](/wiki/Beatmap_Editor/Distance_Snap)
--   [Нестандартные хитсаунды](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides)
--   [Киаи](/wiki/Beatmap_Editor/Kiai_Time)
+- [Меню](Menu) — содержащее в себе множество инструментов и настроек;
+- [AiMod](AiMod) — проверка карты на некоторые распространённые ошибки;
+- [Временная шкала](Timelines), где расположены игровые объекты и текущее положение в песне;
+- [Деления временной шкалы](Beat_Snap_Divisor), влияющие на частоту расстановки объектов;
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/Beatmap_Editor/ru.md]: я там тоже переименую, но потом

File: wiki/Beatmap_Editor/ru.md (hunk 22 above)
Comment (TicClick) [reply wiki/Beatmap_Editor/ru.md]: исправил структуру ради общности, насчёт участков соглашусь с рю.
деления или шаг временной шкалы — это цена деления, как на линейке. от того, что ты нарисуешь там больше черт, больше чисел не появится: они и так существуют, просто скрыты, пока ты их не отметишь

File: wiki/Beatmap_Editor/ru.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (AnnSur) [reply wiki/Beatmap_Editor/ru.md]: Ну, да. Там, в таком списке, в основном предложения начинаются с "The", а так как в русском попроще и нет такого "The", потому в русском переводе можно с маленькой, почему нет. Так даже лучше

**Comment by TicClick** [comment]
I think we're done with the translation polishing, @cl8n could you please take a quick look at what has changed in the original articles, so that I could move forward?

**Comment by cl8n** [comment]
I'll check this one tomorrow if nobody does it before me

**Pull request merged**